src/components/VideoItem.vue

```diff
@@ -23,38 +23,43 @@
                     >Watched</span
                 >
             </div>
-            <p class="uk-text-break">{{ video.title }}</p>
+
+            <div style="display: inline-block; width: 90%">
+                <p class="uk-text-break">{{ video.title }}</p>
+            </div>
+            <div class="uk-align-right" style="margin-left: 0; margin-bottom: 0; display: inline-block; width: 10%">
+                <router-link :to="video.url + '&listen=1'">
+                    <font-awesome-icon icon="headphones"></font-awesome-icon>
+                </router-link>
+            </div>
         </router-link>
 
-        <div :class="{ 'uk-align-left': !(video.views >= 0 || video.uploadedDate) }">
-            <div v-if="video.uploaderUrl && video.uploaderName && !hideChannel">
+        <div v-if="video.uploaderUrl && video.uploaderName && !hideChannel" style="display: flex; flex-flow: row; height: 15%">
+            <img :src="video.uploaderAvatar" loading="lazy" class="uk-border-circle"/>
```

Comment:
A `v-if` or a null check could be added here.

Reply:
I am going to add a v-if, this commit was not the final one.